app/src/main/java/com/rob729/quiethours/Adapter/ProfileListAdapter.kt

```diff
@@ -136,4 +136,13 @@ class ProfileListAdapter(
             }
             .show()
     }
+    fun deleteAll() {
+        val size: Int = profiles.size
+        if (size > 0) {
+            for (i in 0 until size) {
+                removeitem(i)
+                removeWork(profiles[i].profileId.toString())
```

Comment:
Instead of removeitem(i) use notifyItemChanged(i). Put this line after removeWork(profiles[i].profileId.toString())

Reply:
@rob729 on using notifyItemChanged(i) the app isn't functioning as expected. Items are not getting deleting

Reply:
Okay.